packages/app-bridge/src/types/Color.ts

```diff
@@ -134,4 +127,33 @@ export type ColorCreate = {
     alpha: number;
 };
 
-export type ColorPatch = Partial<Omit<Color, 'id' | 'hex' | 'hue' | 'saturation' | 'lightness'>>;
+export type ColorPatch = Partial<Omit<ColorRevision, 'hex' | 'hue' | 'saturation' | 'lightness'>> & {
+    name?: Nullable<string>;
+    sort?: number;
+};
+
+type Rgba = {
```

Comment:
I would suffix Color to these 3 new types: `RgbaColor`, `CmykColor` and `PantoneColor`